REMACS_COMPATIBILITY.md

```diff
@@ -4,13 +4,6 @@ Remacs should generally behave identically to GNU Emacs. If you find a
 difference, please
 [file a bug](https://github.com/Wilfred/remacs/issues/new).
 
-## Differences
-
-   - `base64-encode-string` and `base64-decode-string` does not fail
```

Comment:
Well, for *some* multibyte strings...

The difference is, Emacs only allows encoding ASCII + the Latin1 subset. Characters with codepoint 128-255, which are encoded by two bytes in the multibyte encoding (basically UTF-8), are reduced to one byte before the base64 transformation.

The current Rust implementation just encodes the individual multibyte encoding bytes. That means it can encode any string, but gets different results. That's why the vanilla test had to be changed.

I would argue that being compatible is the right thing to do here.